README.md

```diff
@@ -37,6 +37,8 @@ In the project properties for **pdf-java-toolkit-samples**, select the **Maven**
 
 It will ask if you want to update the product configuration. Click **Yes**.
 
+The samples were developed and tested using [Mars](https://eclipse.org/mars/)
```

Comment:
I wonder if we should mention the version. Personally, I'm up to 4.5.2 (Mars is 4.5, so this is sometimes called Mars.2).

I'm trying to find patch notes to see if there's anything that really affects us.